package.json

```diff
@@ -18,53 +18,23 @@
     "crypto"
   ],
   "scripts": {
-    "prebuild": "rm -rf lib && rm -rf dist",
-    "build": "yarn run build:node && yarn run build:web && yarn run build:types",
-    "bundle": "webpack --config webpack.bundle.config.js",
-    "build:node": "babel --no-babelrc --config-file ./babel.config.json src --copy-files -x .ts -d dist --source-maps",
-    "build:web": "babel --no-babelrc --config-file ./babel.web.config.json src --copy-files -x .ts -d lib --source-maps",
-    "build:types": "tsc --declaration --outDir dist --emitDeclarationOnly",
-    "proto:gen": "pbjs -t static-module -o ./src/proto/payload.js ./src/proto/payload.proto && pbts -o ./src/proto/payload.d.ts ./src/proto/payload.js && yarn run lint --fix",
-    "check-types": "tsc --noEmit",
-    "lint": "eslint --ext .ts src/",
+    "build": "aegir build --ts",
+    "lint": "aegir lint --ts",
     "pretest": "yarn check-types",
-    "test": "yarn run test:node && yarn run test:web",
-    "test:node": "mocha -r ./babel-register.js \"test/**/*.test.ts\"",
-    "test:web": "karma start"
+    "test": "aegir test --ts",
+    "test:node": "aegir test -t node --ts",
+    "test:browser": "aegir test -t browser --ts",
+    "check-types": "tsc --noEmit",
+    "proto:gen": "pbjs -t static-module -o ./src/proto/payload.js ./src/proto/payload.proto && pbts -o ./src/proto/payload.d.ts ./src/proto/payload.js && yarn run lint --fix"
```

Comment:
I don't think this would work in packaged library. tsc doesn't copy non ts files. We could either enable allowJs in tsconfig.json or add npm script to copy it manually in postbuild task